src/main/java/com/example/module2/StudentMain.java

```diff
@@ -3,122 +3,152 @@
         student4.setDateOfBirth(birthDate4);
         student4.setAddress(address4);
-        student4.setCourse(Course.THIRD);
+        student4.setCourse(Course.FIFTH);
         student4.setFaculty(Faculty.FILM);
-        student4.setGroup(Group.ONE);
-        System.out.println(student4.toString());
+        student4.setGroup(Group.TWO);
+        System.out.println(student4);
 
         Date birthDate3 = format.parse(date3);
         student3.setDateOfBirth(birthDate3);
         student3.setAddress(address3);
-        student3.setCourse(Course.FIFTH);
-        student3.setFaculty(Faculty.FILM);
-        student3.setGroup(Group.TWO);
-        System.out.println(student3.toString());
+        student3.setCourse(Course.SIXTH);
+        student3.setFaculty(Faculty.ART);
+        student3.setGroup(Group.ONE);
+        System.out.println(student3);
 
         Date birthDate2 = format.parse(date2);
         student2.setDateOfBirth(birthDate2);
         student2.setAddress(address2);
-        student2.setCourse(Course.SIXTH);
-        student2.setFaculty(Faculty.ART);
-        student2.setGroup(Group.ONE);
-        System.out.println(student2.toString());
+        student2.setCourse(Course.FIFTH);
+        student2.setFaculty(Faculty.FILM);
+        student2.setGroup(Group.THREE);
+        System.out.println(student2);
 
         Date birthDate1 = format.parse(date1);
         student1.setDateOfBirth(birthDate1);
         student1.setAddress(address1);
-        student1.setCourse(Course.FIFTH);
-        student1.setFaculty(Faculty.FILM);
+        student1.setCourse(Course.SECOND);
         student1.setGroup(Group.THREE);
-        System.out.println(student1.toString());
-
-        Date birthDate = format.parse(date);
-        student.setDateOfBirth(birthDate);
-        student.setAddress(address);
-        student.setCourse(Course.SECOND);
-        student.setGroup(Group.THREE);
-        student.setFaculty(Faculty.DESIGN);
-        System.out.println(student.toString());
-
-//a) список студентов заданного факультета;
-        List<Student> studentList = new ArrayList<>();
-        studentList.add(student);
-        studentList.add(student1);
-        studentList.add(student2);
-        studentList.add(student3);
-        studentList.add(student4);
-        studentList.add(student5);
-        List<Student> filmFaculty = new ArrayList<>();
-        for (Student equalsFaculty : studentList) {
-            if (equalsFaculty.getFaculty() == Faculty.FILM) {
-                filmFaculty.add(equalsFaculty);
+        student1.setFaculty(Faculty.DESIGN);
+        System.out.println(student1);
+
+        return Arrays.asList(student1, student2, student3, student4, student5, student6);
+    }
+
+    private final static SimpleDateFormat format = new SimpleDateFormat("dd.MM.yyyy");
+
+    //a) список студентов заданного факультета;
+    private static List<Student> printFilmFacultyStudents(List<Student> studentList) {
```

Comment:
This method returns List, so its name doesn't correspond its behavior.
Please change name to smth like
`findFilmFacultyStudents`

And also remove list output to console from this method.